src/Components/Navbar/Navbar.jsx

```diff
@@ -1,9 +1,15 @@
 import { NavLink } from "react-router-dom";
-import {FaUserCircle, AiFillHeart, FaShoppingCart, GiHamburgerMenu} from "../../assets/icons";
+import {AiFillHeart, FaShoppingCart, GiHamburgerMenu} from "../../assets/icons";
+import { useAuth } from "../../Context/auth.context";
+import { useCart } from "../../Context/cart.context";
+import { useWishlist } from "../../Context/wishlist.context";
```

Comment:
You can import all these context from single index.js file

Reply:
Done